Sources/PackageCollections/JSONModel/JSONCollection+v1.swift

```diff
@@ -0,0 +1,262 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import struct Foundation.Date
+import struct Foundation.URL
+
+import PackageModel
+
+extension JSONPackageCollectionModel {
+    public enum V1 {}
+}
+
+extension JSONPackageCollectionModel.V1 {
+    public struct PackageCollection: Equatable, Codable {
+        /// The name of the package collection, for display purposes only.
+        public let name: String
+
+        /// A description of the package collection.
+        public let _description: String?
+
+        /// An array of keywords that the collection is associated with.
+        public let keywords: [String]?
```

Comment:
Is this optional so it doesn't get coded at all if empty, or is there a semantic difference?  (noting for example that `packages` below isn't optional — though admittedly we don't expect to have no packages the way there might be no keywords)

Reply:
It is optional so it doesn't get coded if `nil` and deserialization doesn't fail if the key is missing. `[]` still gets coded.

`packages` is required. Decode will fail if the key is missing or value is `nil`. `[]` will get through and we will probably need validation to trap that case.

Reply:
The distinction between optional and required arrays makes a lot of sense.  Thanks for explaining!